pkg/controller/certificates/cleaner/cleaner.go

```diff
@@ -177,7 +177,7 @@ func isFailedPastDeadline(csr *capi.CertificateSigningRequest) bool {
 func isIssuedPastDeadline(csr *capi.CertificateSigningRequest) bool {
 	for _, c := range csr.Status.Conditions {
 		if c.Type == capi.CertificateApproved && isIssued(csr) && isOlderThan(c.LastUpdateTime, approvedExpiration) {
-			klog.Infof("Cleaning CSR %q as it is more than %v old and approved.", csr.Name, approvedExpiration)
+			klog.InfoS("Cleaning CSR as it is past expiration and approved.", "csr", klog.KObj(csr), "approvedExpiration", approvedExpiration)
```

Comment:
```suggestion
		klog.InfoS("Cleaning CSR as it is past pending expiration and unhandled.", "csr", klog.KObj(csr), "age", time.Since(csr.CreationTime), "expiration": pendingExpiration, "condition", capi.CertificateApproved)
```

Reply:
I didn't use `"age", time.Since(csr.CreationTime)` as it throws this error: `csr.CreationTime undefined (type *"k8s.io/api/certificates/v1".CertificateSigningRequest has no field or method CreationTime).`